src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +206,16 @@ func runClientProxy(cfg *config.Config) {
 		exit(err)
 	}
 
+	if *clearProxySettings {
+		// This is a workaround that attempts to fix a Windows-only problem where
+		// Lantern was unable to clean the system's proxy settings before logging
+		// off.
+		//
+		// See: https://github.com/getlantern/lantern/issues/2776
+		doPACOff("http://" + cfg.UIAddr + "/proxy_on.pac")
```

Comment:
We have a check below whether ``cfg.UIAddr`` is supplied or not (lines 227-229) that we probably want to come before this block, right?